src/components/quiz-question/quiz-question-wrapper.scss

```diff
@@ -1,2 +1,64 @@
-.quizQuestionWrapper {
+@import '../../theme.scss';
+
+.ivqQuestionContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100%;
+  padding: 32px 32px 64px 32px;
+
+  .ivqButtonsWrapper {
+    display: flex;
+    width: 100%;
+    margin-top: 16px;
+    .continueButton {
+      @include ivq-primary-button;
+      margin-right: 8px;
+    }
+    .skipButton {
+      @include ivq-secondary-button;
+    }
+  }
+}
+
+.ivqBottomBar {
+  &.ivqSeekBarWrapper {
+    visibility: visible;
+    max-height: 56px;
+    bottom: 32px;
+    transition: none;
+  }
+
+  .ivqNavigationWrapper {
+    width: 100%;
+    height: 38px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    .navigationButton {
+      background: transparent;
+      padding: 0;
+      margin: 0 12px;
+      border: none;
+      width: 20px;
+      height: 32px;
+      text-align: center;
+      &:hover:not(.disabled) {
+        opacity: 0.9;
+        cursor: pointer;
+      }
+      &.disabled {
+        opacity: 0.4;
+        cursor: not-allowed;
```

Comment:
please remove. It is too much :) Just make sure cursor doesn't change to pointer on hover